blockchain/standalone/treasury.go

```diff
@@ -24,26 +24,29 @@ func IsTreasuryVoteInterval(height, tvi uint64) bool {
 	return height%tvi == 0 && height != 0
 }
 
-// CalculateTSpendWindowStart calculates the start of a treasury voting window
-// based on the parameters that are passed. Great care must be taken to ensure
-// this function is only called with an expiry that *IS* on a TVI.
-func CalculateTSpendWindowStart(expiry uint32, tvi, multiplier uint64) (uint32, error) {
-	if !IsTreasuryVoteInterval(uint64(expiry-2), tvi) {
-		return 0, ruleError(ErrTSpendStartInvalidExpiry,
-			fmt.Sprintf("invalid start expiry: %v", expiry))
+// CalcTSpendWindow calculates the start and end of a treasury voting window
+// based on the parameters that are passed.  An error will be returned if the
+// provided expiry is not two more than a treasury vote interval (TVI) or before
+// a single voting window is possible.
+func CalcTSpendWindow(expiry uint32, tvi, multiplier uint64) (uint32, uint32, error) {
```

Comment:
Personal preference, so feel free to ignore, but I would like it if the return values were named, since they are the same type next to each other. i.e. `(start, end uint32, err error)`

Reply:
I can change it if people really think it's necessary, but I generally make it a point to avoid named returns in Go because they have historically resulted in a lot of bugs due to the fact they can be updated with intermediate states and end up returning non-zero values even in the case of error.

Reply:
In that case, PLEASE IGNORE!